homeassistant/components/media_source/const.py

```diff
@@ -16,5 +16,5 @@
 }
 URI_SCHEME = "media-source://"
 URI_SCHEME_REGEX = re.compile(
-    r"^media-source:\/\/(?:(?P<domain>(?!.+__)(?!_)[\da-z_]+(?<!_))(?:\/(?P<identifier>(?!\/).+))?)?$"
```

Comment:
Talked with @frenck about this change. The negative look ahead for double underscore was breaking because the URL returned by Xbox for the gameclip media has `&__gda__=` as a part of it. I removed it because though domains are not allowed to have `__`, they are pretty protected already by code review.